ut/s3_delete_multiple_objects_action_test.cc

```diff
@@ -845,3 +846,46 @@ TEST_F(S3DeleteMultipleObjectsResponseBodyTest, AllSucessQuietModeOff) {
   // Verify 'response_xml' matches 'expect_xml'
   EXPECT_STREQ(expect_xml.c_str(), response_xml.c_str());
 }
+
+TEST_F(S3DeleteMultipleObjectsActionTest, DelayedDeleteMultipleObjects) {
+
+  S3Option::get_instance()->set_s3server_obj_delayed_del_enabled(true);
+
+  action_under_test->motr_kv_writer =
+      motr_kvs_writer_factory->mock_motr_kvs_writer;
+  action_under_test->motr_writer = motr_writer_factory->mock_motr_writer;
+
+  EXPECT_CALL(*(motr_writer_factory->mock_motr_writer),
+              delete_objects(_, _, _, _)).Times(0);
+
+  action_under_test->cleanup();
+}
+
+TEST_F(S3DeleteMultipleObjectsActionTest, SizeBucketingOfObjects) {
```

Comment:
you dont have to have this tests multiple times, if there is test file for Utilities then add to that, else in one place its good enough.